.vscode/launch.json

```diff
@@ -23,12 +23,14 @@
             "request": "launch",
             "runtimeExecutable": "${execPath}",
             "args": [
-                "--extensionDevelopmentPath=${workspaceFolder}"
+                "--extensionDevelopmentPath=${workspaceFolder}",
+                "--verbose"
             ],
             "outFiles": [
                 "${workspaceFolder}/out/**/*.js"
             ],
-            "preLaunchTask": "npm: watch"
+            "preLaunchTask": "npm: build",
```

Comment:
Noticed this was changed from "watch" to "build". Could Webpack watch task could be used here?

Reply:
@tjohnsonBCM Addressed in https://github.com/zowe/vscode-extension-for-zowe/pull/860/commits/bf491f5135f8441fc72c2c3c2e48fdd7555dd93c, based on [CPPTools'](https://github.com/microsoft/vscode-cpptools/tree/master/Extension) configuration for extension launch in watch and development mode.